### PR TITLE
2017 04 form wizard

### PR DESCRIPTION
**Pull request by MagdaN**
@slomo @fuzzylogic2000 I played around with the form wizard and it seems to work quite nicely. Feel free to keep an going in this PR or merge so that we all can keep on working on different parts.

### CODE REVIEW

**Comment by MagdaN** [comment]
@slomo @fuzzylogic2000 we have a number of charfields where would like to enable multiple choices but don't want to use a m2m relation - I was thinking about  adding a custom widget but found [this](https://github.com/goinnn/django-multiselectfield) which works quite nicely. So should we use this?

File: advocate_europe/templates/formtools/wizard/wizard_form.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            <div class="col-md-6 form-section_title">
+                <h1>{{ form.section_name }}</h1>
+            </div>
+        </div>
+
+
+        {% csrf_token %}
+        {{ wizard.form.media }}
+        {{ wizard.management_form }}
+        {% if wizard.form.forms %}
```
Comment (slomo) [inline advocate_europe/templates/formtools/wizard/wizard_form.html:20]: Where is that needed? Is that for MultiForms?

**Comment by slomo** [comment]
Nice that so much functionality can be archived with so little code.

**Comment by slomo** [comment]
I think we should try the library mentioned by @MagdaN in another PR.

File: advocate_europe/templates/formtools/wizard/wizard_form.html (hunk 1 above)
Comment (MagdaN) [reply advocate_europe/templates/formtools/wizard/wizard_form.html:20]: I am not sure. I simply overwrote this template here: https://github.com/django/django-formtools/blob/master/formtools/templates/formtools/wizard/wizard_form.html

**Pull request merged**